### PR TITLE
Added Recent Talks to Meshery.io site

### LINKED ISSUE #1396: Add recent talks into meshery.io talks and presentations page
#### Current State of the Meshery.io site

Talks And Presentations can be found at https://meshery.io/talks-and-presentations

#### Desired State of the Meshery.io site

Navigate to https://layer5.io/community/events/2 and access the Events section. 
 - Screenshot 2023-09-01 010047
Incorporate the ten most recent <b>Talks</b> into the https://meshery.io/talks-and-presentations page.

<h3><details><summary>Click to get the ten most recent Talks</summary>
<ol>
<li><a href="https://layer5.io/community/events/kubecon-cloudnativecon-north-america-chicago-2023">https://layer5.io/community/events/kubecon-cloudnativecon-north-america-chicago-2023</a></li>
<li><a href="https://layer5.io/community/events/istiocon-virtual-2023">https://layer5.io/community/events/istiocon-virtual-2023</a></li>
<li><a href="https://layer5.io/community/events/kcd-chennai-2023">https://layer5.io/community/events/kcd-chennai-2023</a></li>
<li><a href="https://layer5.io/community/events/may-maintainers-circle-2023">https://layer5.io/community/events/may-maintainers-circle-2023</a></li>
<li><a href="https://layer5.io/community/events/kubecon-cloudnativecon-eu-amsterdam-2023">https://layer5.io/community/events/kubecon-cloudnativecon-eu-amsterdam-2023</a></li>
<li><a href="https://layer5.io/community/events/kubecon-cloudnative-north-america-2022">https://layer5.io/community/events/kubecon-cloudnative-north-america-2022</a></li>
<li><a href="https://layer5.io/community/events/hacktoberfest-prep-easing-into-cncf-open-source-projects">https://layer5.io/community/events/hacktoberfest-prep-easing-into-cncf-open-source-projects3</a></li>
<li><a href="https://layer5.io/community/events/devops-and-docker-live-show">https://layer5.io/community/events/devops-and-docker-live-show</a></li>
<li><a href="https://layer5.io/community/events/open-source-summit-north-america-2022">https://layer5.io/community/events/open-source-summit-north-america-2022</a></li>
<li><a href="https://layer5.io/community/events/kubecon-cloudnativecon-eu-virtual-2022">https://layer5.io/community/events/kubecon-cloudnativecon-eu-virtual-2022</a></li>
</ol>
</details></h3>

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by akhilender-bongirwar**
**Description**
This commit addresses the issue by incorporating the ten most recent Talks from https://layer5.io/community/events/2 into the Meshery.io Talks and Presentations page at https://meshery.io/talks-and-presentations. Users can now easily access and stay updated with the latest talks and presentations related to Meshery on our website.

This PR fixes #1396 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
hey @akhilender-bongirwar 

Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)
https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Review by abhijeetgauravm — APPROVED** [review]
LGTM!

**Comment by sudhanshutech** [comment]
@akhilender-bongirwar can you also add the latest hactoberfest topic in talks

**Comment by akhilender-bongirwar** [comment]
Hey @sudhanshutech, I just noticed that I already added the hacktober fest in the talks and presentation section already.

Screenshot 2023-09-25 235532
Screenshot 2023-09-25 235616

If it is not this please let me know : ) .

**Comment by abhijeetgauravm** [comment]
> Hey @sudhanshutech, I just noticed that I already added the hacktober fest in the talks and presentation section.
> 
> Screenshot 2023-09-25 235532 Screenshot 2023-09-25 235616
> 
> If it is not this please let me know : ) .

Latest one u can find [here](https://layer5.io/community/events/hacktoberfest-prep-2023-easing-into-cncf-open-source-projects) @akhilender-bongirwar

**Comment by akhilender-bongirwar** [comment]
@abhijeetgauravm, Thank you.

**Review by abhijeetgauravm — APPROVED** [review]
Looks good!

**Comment by abhijeetgauravm** [comment]
Let's discuss this on website call today at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima)

**Comment by akhilender-bongirwar** [comment]
Sure @abhijeetgauravm, I will add this : ).

File: _data/talks.yml [hunk 1]
```diff
@@ -1,5 +1,106 @@
-
 # Meshery Related Talks/Presentations
+- title: "Hacktoberfest Prep 2023: Easing into CNCF Open Source Projects"
```
Comment (vishalvivekm) [inline _data/talks.yml:2]:
```suggestion
- title: "Hacktoberfest Prep 2023: Easing into CNCF Open Source Projects"
  video: https://www.youtube.com/live/T7xLUqrnxtI?si=a_TcRr_x3v95oqWH
```

File: _data/talks.yml [hunk 2]
```diff
@@ -1,5 +1,107 @@
...
+  presenter2: Nate Waddington
+
+- title: KubeCon + CloudNativeCon EU Amsterdam 2023
+  presentation: https://layer5.io/community/events/kubecon-cloudnativecon-eu-amsterdam-2023
+  presenter1: Lee Calcote
+  presenter1_profile: https://layer5.io/community/members/lee-calcote
+  new_window: true
+
+- title: KubeCon + CloudNative North America 2022
+  presentation: https://layer5.io/community/events/kubecon-cloudnative-north-america-2022
```
Comment (vishalvivekm) [inline _data/talks.yml:60]:
```suggestion
  presentation: https://layer5.io/community/events/kubecon-cloudnative-north-america-2022
  presenter1: Layer5 Team
  presenter1_profile: https://layer5.io/company/about
```

**Review by vishalvivekm — COMMENTED** [review]
can add `Layer5 Team` as presenter here

File: _data/talks.yml [hunk 3]
```diff
@@ -1,5 +1,107 @@
-
 # Meshery Related Talks/Presentations
+- title: "Hacktoberfest Prep 2023: Easing into CNCF Open Source Projects"
+  video: https://www.youtube.com/live/T7xLUqrnxtI?si=a_TcRr_x3v95oqWH
```
Comment (vishalvivekm) [inline _data/talks.yml:3]:
```suggestion
  video: https://www.youtube.com/live/T7xLUqrnxtI?si=a_TcRr_x3v95oqWH
  embeded_video: https://www.youtube.com/embed/T7xLUqrnxtI?si=OEW5miLK3XyLZh3p
```

**Review by vishalvivekm — APPROVED** [review]
Thanks for this @akhilender-bongirwar.
LGTM.

**Pull request merged**